.github/workflows/ubuntu.yml

```diff
@@ -1,8 +1,10 @@
-name: nginx-otel-module-check
-run-name: ${{ github.actor }} is triggering pipeline
+name: Ubuntu build
+
 on:
   push:
-  workflow_dispatch:
```

Comment:
Why do you remove this?

Reply:
I just don't see how it's useful. If it is, we can keep it.